src/Form/FieldTextarea.js

```diff
@@ -0,0 +1,42 @@
+import classNames from 'classnames';
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
+
+import FieldInput from './FieldInput';
+import IconEdit from './icons/IconEdit';
+
+export default class FieldTextarea extends FieldInput {
+  getInputElement(attributes) {
+    let {props} = this;
+
+    let classes = classNames(props.inputClass, props.sharedClass);
+    attributes = this.bindEvents(attributes);
+
+    if (this.isEditing() || props.writeType === 'input') {
+      return (
```

Comment:
Also aren't we repeating FieldInput here?
Could we make it a bit more flexible by adding the option of rendering different tags (when editing) for a core text field component and then extending that to create a FieldInput and FieldTextarea.
Just an idea